lardataalg/DetectorInfo/DetectorPropertiesStandard.h

```diff
@@ -121,6 +121,8 @@ namespace detinfo {
                 "between the wire planes. This is appropriate for "
                 "recob::RawDigits, and recob::Wires from the 1D unfolding, "
                 "but is not appropriate for recob::Wires from WireCell. "
+                "When disabled, the conversion propgates to the last plane, "
```

Comment:
Spelling change: "propgates" -> "propagates"

Reply:
Got it.  Thank you @knoepfel :)